app/src/main/java/com/framgia/soundclound/util/BindingUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.framgia.soundclound.util;
+
+import android.databinding.BindingAdapter;
+import android.graphics.PorterDuff;
+import android.support.design.widget.TabLayout;
+import android.support.v4.content.ContextCompat;
```

Comment:
ERROR: Unused import: android.support.v4.content.ContextCompat. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports